src/Api/Dto/Event.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Api\Dto;
+
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Get;
+use App\Api\State\EventRepresentationProvider;
+
+#[ApiResource(operations: [
+    new Get(
+        openapiContext: [
+            'summary' => 'Get single event base on identifier',
+            'parameters' => [
+                [
+                    'name' => 'id',
+                    'in' => 'path',
+                    'required' => true,
+                    'schema' => [
+                        'type' => 'integer',
+                    ],
+                ],
+            ],
+        ],
+        output: EventRepresentationProvider::class,
+        provider: EventRepresentationProvider::class,
+    ),
+])]
+class Event
+{
+    #[ApiProperty(
+        identifier: true,
+    )]
+    private int $id;
+
+    #[ApiProperty()]
+    private string $title;
+    #[ApiProperty()]
+    private string $excerpt;
+    #[ApiProperty()]
+    private string $description;
+    #[ApiProperty()]
+    private string $url;
+    #[ApiProperty()]
+    private string $ticketUrl;
+    #[ApiProperty()]
+    private bool $public;
```

Comment:
Consider adding blank lines between properties for added readability:

```suggestion
    #[ApiProperty()]
    private string $title;
    
    #[ApiProperty()]
    private string $excerpt;
    
    #[ApiProperty()]
    private string $description;
    
    #[ApiProperty()]
    private string $url;
    
    #[ApiProperty()]
    private string $ticketUrl;
    
    #[ApiProperty()]
    private bool $public;
```